src/controllers/article.controller.js

```diff
@@ -7,47 +9,6 @@ import Pagination from '../helpers/pagination';
  * @class UserController
  */
 class ArticleController {
-  /**
-   * Get users and their corresponding profiles
-   * @async
-   * @param {object} req - Request object
```

Comment:
Hello could you rebase again so it does not seem as though you removed  all the code in this module

Reply:
ok